src/patternfly/components/ProgressStepper/progress-stepper.scss

```diff
@@ -0,0 +1,248 @@
+.pf-c-progress-stepper {
+  // Stepper variables
+  --pf-c-progress-stepper--GridAutoFlow: column;
+  --pf-c-progress-stepper--GridTemplateRows: auto 1fr;
+  --pf-c-progress-stepper--GridTemplateColumns: initial;
+  --pf-c-progress-stepper--m-vertical--GridAutoFlow: row;
+  --pf-c-progress-stepper--m-vertical--GridTemplateColumns: auto 1fr;
+  --pf-c-progress-stepper--m-compact--GridAutoFlow: row;
+  --pf-c-progress-stepper--m-compact--GridTemplateColumns: repeat(auto-fill, 28px);
+
+  // Step variables
+
+  // Ste -connector variables
+  --pf-c-progress-stepper__step-connector--JustifyContent: start;
+  --pf-c-progress-stepper--m-center__step-connector--JustifyContent: center;
+
+  // Step icon variables
+  --pf-c-progress-stepper__step-icon--Width: 28px;
+  --pf-c-progress-stepper__step-icon--Height: var(--pf-c-progress-stepper__step-icon--Width);
+  --pf-c-progress-stepper__step-icon--FontSize: var(--pf-global--icon--FontSize--md);
+  --pf-c-progress-stepper__step-icon--Color: var(--pf-global--Color-100);
+  --pf-c-progress-stepper__step-icon--BackgroundColor: var(--pf-global--BackgroundColor--light-200);
+  --pf-c-progress-stepper__step-icon--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-progress-stepper--m-compact__step-icon--Width: 18px;
+  --pf-c-progress-stepper--m-compact__step-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+
+  // Icon adjustments to fix differences in alignment
+  --pf-c-progress-stepper__pficon--MarginTop: 3px;
+  --pf-c-progress-stepper__fa-exclamation-triangle--MarginTop: -5px;
+  --pf-c-progress-stepper--m-compact__pficon--MarginTop: 2px;
+  --pf-c-progress-stepper--m-compact__fa-exclamation-triangle--MarginTop: -3px;
+
+
+  // Step :before (the connecting line) variables
+  --pf-c-progress-stepper__step-connector--before--Top: calc(var(--pf-c-progress-stepper__step-icon--Height) / 2);
```

Comment:
Since it's a 2px border, it could also be brought up `transform: translateY(-50%)` or add `- var(--pf-global--BorderWidth--md) / 2)` to this calc. Same for the vertical before.

Reply:
Good catch